core/shim.js

```diff
@@ -1,88 +1,88 @@
 exports.createPlatformModule = function(platform) {
-	if (!platform.sendMessage) {
-		platform.sendMessage = function() {
-			throw 'What kind of shit platform is this that doesn\'t even support sending messages?';
-		};
-	}
+    if (!platform.sendMessage) {
+        platform.sendMessage = function() {
+            throw 'What kind of shit platform is this that doesn\'t even support sending messages?';
+        };
+    }
 
-	if (!platform.sendUrl) {
-		platform.sendUrl = function(url, thread) {
-			platform.sendMessage(url, thread); // fallback to sending a message
-		};
-	}
+    if (!platform.sendUrl) {
+        platform.sendUrl = function(url, thread) {
+            platform.sendMessage(url, thread); // fallback to sending a message
+        };
+    }
 
-	if (!platform.sendImage) {
-		platform.sendImage = function(type, image, description, thread) {
-			switch(type) {
-				case 'url': // fallback to sending a url
-					platform.sendMessage(description, thread);
-					platform.sendUrl(image, thread);
-					break;
-				case 'file': // fallback to sending a file
-					platform.sendFile(image, thread);
-					break;
-				default: // fallback to sending a message
-					platform.sendMessage(description, thread);
-					platform.sendMessage('I also have something to send you but cant seem to do so...', thread);
-					break;
-			}
-		};
-	}
+    if (!platform.sendImage) {
+        platform.sendImage = function(type, image, description, thread) {
+            switch(type) {
+                case 'url': // fallback to sending a url
+                    platform.sendMessage(description, thread);
+                    platform.sendUrl(image, thread);
+                    break;
+                case 'file': // fallback to sending a file
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 space characters but found 16.](https://www.codacy.com/app/theprojectshop/Kassy/file/2588679405/issues/source?bid=3282987&fileBranchId=3291254#l21)